core/modules/widgets/fieldmangler.js

```diff
@@ -135,6 +135,10 @@ FieldManglerWidget.prototype.handleAddTagEvent = function(event) {
 			$tw.utils.pushTop(modification.tags,tag);
 			this.wiki.addTiddler(new $tw.Tiddler(tiddler,modification));			
 		}
+	} else if(typeof event.param === "string" && event.param.trim() !== "" && this.mangleTitle.trim() !== "") {
```

Comment:
I am not sure if it is necessary or not, but both are checked in the case where a tiddler does exist. The title is checked indirectly because getTiddler returns undefined if the name is whitespace or an empty string.
I am not sure if adding an outer if statement that checks to make sure that there is a valid string for the tag before checking if the tiddler exists is better than having that in each of the two if statements the way it is now.